crates/sui-indexer/src/main.rs

```diff
@@ -2,14 +2,15 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use clap::Parser;
-use tracing::info;
+use tracing::{info, warn};
 
 use sui_indexer::errors::IndexerError;
 use sui_indexer::metrics::start_prometheus_server;
 use sui_indexer::IndexerConfig;
 
 #[tokio::main]
 async fn main() -> Result<(), IndexerError> {
+    warn!("WARNING: Sui indexer is still experimental and we expect occasional breaking changes that require backfills.");
```

Comment:
This line needs to be below when the tracing infrastructure is setup otherwise it won't be output.